42mad

```diff
@@ -416,7 +417,7 @@ else
         pdauth="$(awk 'NR==2{print $1}' "$usbfile")"
         check_session
         origin=$(/system/bin/curl -s -k -L --user "$pdauth" "${pdserver}/autoconfig/${session_id}/origin")
-        log_msg 2 "Hello, this is 42mad from $origin!"
+        log_msg 2 "Hello, this is 42mad from $origin! My current connection and IP is $ip"
```

Comment:
for readability i think we can remove "connection and"
leaving it as: "... My current IP is $ip"

Reply:
> for readability i think we can remove "connection and"
> leaving it as: "... My current IP is $ip"

OK. Makes sense now that connection type is no longer present.

Changes made and added another IP check / log message after (possible) MAC change. Not checking if there was an actual change of IP (there is no need) just posting address again, ether it changed or not.